GetRecd/Controllers/SignInViewController.swift

```diff
@@ -103,6 +103,10 @@ class SignInViewController: AuthenticationViewController, UITextFieldDelegate {
         }
     }
 
+    @IBAction func googleSignInButtonPressed(_ sender: Any) {
```

Comment:
DIdn't this function already exist? What changed here?

Reply:
Not sure, it showed that it didn't exist for some reason